.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+	"name": "libchewing-devcontainer",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"build": {
+		"dockerfile": "./Dockerfile"
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"eamodio.gitlens", // IDE Git information
+				"shardulm94.trailing-spaces", // Show trailing spaces
+				"christian-kohler.path-intellisense",
+				"ms-azuretools.vscode-docker",
+				"ms-vscode.cmake-tools", // Cmake
+				// Rust language support
+				"rust-lang.rust", // Use "rust-lang.rust-analyzer" once MSRV reaches 1.64
```

Comment:
If we use `rust-lang.rust-analyzer` should it be able to download the corresponding pre-built version automatically?

Reply:
25208bfe